run.c

```diff
@@ -267,12 +266,13 @@ void find_next_sync_point_and_drift(int with_fof)
 
         set_global_time(nexttime);
 
-        move_particles(All.Ti_nextoutput);
-
+        /* Do domain decomp and rebuild tree before doing
+         * power spectrum or FoF; if the particles have
+         * moved a long way, performance will be degraded.*/
```

Comment:
Yes - without doing this the neutrinos will break. Do you want me to add "without doing this" to the comment?